Sharecare/cordova-plugin-facebook-connect
# feat: CLARITY-950 Upgrade iOS SDK to 15 and Android SDK 13

To test:
- download repo `git clone git@github.com:Sharecare/cordova-plugin-facebook-connect.git`
- checkout this branch `git checkout feat/ios-15-android-13`
- on Clarity, install this repo locally `npm install ../cordova-plugin-facebook-connect`
- on Clarity, run iOS with `npm run ern:livereload-ios`
- Open Facebook Event Tester: https://www.facebook.com/events_manager2/list/app/193644072681742/test_events?act=275388973119245
- open Facebook App in your iPhone
- open ERN App
- you should see events coming through

<img width="1311" alt="image" src="https://user-images.githubusercontent.com/3648665/225716603-b920bdc4-1d74-4244-afae-98b0496822a3.png">


Also, advanced settings should show the correct versions:
https://developers.facebook.com/apps/193644072681742/settings/advanced/?business_id=700888080043868

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/3648665/225717471-82b42789-7378-480e-8a4f-b3f6656f0af6.png">
